package.json

```diff
@@ -26,10 +26,10 @@
     "fs-extra": "^0.30.0",
     "glob": "^7.0.0",
     "lodash": "^4.5.1",
-    "loopback": "^2.0.0",
+    "loopback": "^3.23.2",
     "loopback-boot": "^2.6.5",
     "loopback-component-explorer": "^6.0.1",
-    "loopback-datasource-juggler": "^2.27.0",
+    "loopback-connector-mysql": "^5.3.1",
```

Comment:
The workspace API & implementation does not use MySQL database in any means, please remove this dependency.

We do access MySQL database in tests, the MySQL connector is installed by the test suite - no need to do it here.